assets/models/Iceshard1.tscn

```diff
@@ -1,8 +1,26 @@
-[gd_scene load_steps=2 format=2]
+[gd_scene load_steps=4 format=2]
 
 [ext_resource path="res://assets/models/Iceshard1.mesh" type="ArrayMesh" id=1]
 
+[sub_resource type="Shader" id=1]
```

Comment:
Shader needs to be in its own file which could then be hooked up to the iceshard's ShaderMaterial (this needs to be created anew as the iceshard currently still use a SpatialMaterial)

Reply:
Agreed. I'll do my own review on this PR once this is resolved.